precipitation-uploader.py

```diff
@@ -0,0 +1,119 @@
+from dotenv import load_dotenv
+import requests
+import json
+import pandas as pd
+import numpy as np
+import os
+from functools import partial
+
+load_dotenv()
+
+csv_path =os.getenv('precip_filename')
+csv_delimiter = ","
+ckan_api_url = os.getenv('ckan_url')
+api_key =  os.getenv('api_keys')
+resource_id = os.getenv('precip_resource_id')
+
+def datastore_delete(ckan_api_url, resource_id, api_key):
```

Comment:
Method seems to be repeated between several files. Should be extracted to helper file and imported.